### ISSUE TITLE
Epic > 020 > Parent: New Splash Screen of www.villagetronic.com

### ISSUE THREAD

**Issue by VillageHubertChen** [V-Squared/V-Squared.github.io#186]
# Problem

The splash screen on http://www.villagetronic.com is still promoting the old Indiegogo campaign.
# Proposal for text for new screen
### New ViPC

We are developing a high end portable Workstation PC, ready for order in summer this year. Based on second generation ViCase. Key features:
- The first Open Modular Mini PC Workstation
- Small & portable Mini PC design.
- Fluid Form Factor enable changing a Mini PC to All-In-One and vice versa
- Many CPU choices up to i7-6700
- Ultra fast disks with PCI Express NVME boot disks and SSD RAID data disks
- Many GPU choices up to K1200
- Many ready build models available or you can go fully custom build
- Very quiet 3 chamber airflow design, yet cooling capacity of 300W
- Easy upgrading, easy maintenance, easy repair.
- Build in Surge Protection, EMI Filter and Power Distribution for displays 
### New ViCase

ViCase 2G is a high end Workstation DIY PC Case. Key Features:
- Very high quality. CNC cut from aircraft grade aluminum, sandblasted and anodized
- 10 Year Warranty
- Compatible to Desktop Standard Parts, e.g. Mini ITX Mainboard and Flex ATX PSU
- Optimized three chamber airflow design with cooling capacity 300W
- Double Bracket, Low Profile, 75W PCI Express Slot
- Small, portable dimensions: 245 x 245 x 105 mm³ = 6 Liters
- Washable magnetic quick snap filters
- Build in Surge Protection, EMI Filter and Power Distribution for displays 
- Easy access from all sides during assembly via removable sides
### New V² Community Development Site
- Anyone can file Bug Reports or Feature Ideas via V²Flow
- Anyone can improve articles or even contribute complete new ones via V²Flow
- We are currently focusing all our efforts on two sections of the V²Site
  - V²Plan
  - V²DIY
### A new page every day

We are now publishing a new page every day. We will announce it on our Facebook Page. Your feedback is most welcome.
# Task Lists
- [x] HC: Write proposal new Splash Screen
- [x] ML: Render and upload new Image proposal
- [x] ML: Final rendering of new Image
- [x] HC: Processing and uploading of [Final Image](#issuecomment-227924022)
- [ ] BC: Edit HC's writing
# What needs to be finished first:
- [ ] V²Flow Button and Article
- [ ] Basic Home Page
- [x] HC: → [True North Page](https://v-squared.github.io/plan/true-north/)
- [ ] V²Agile 7 Layer Image
- [ ] BC: [Workstation Comparison Article](https://v-squared.github.io/diy/compare/workstation-comparison/)
- [x] BC: → [V²DIY Intro Page](https://v-squared.github.io/diy/)
- [ ] BC: Finish writing of text for this initial post
- [ ] V²Plan Intro Page

**Comment by VillageHubertChen** [comment]
# New Image Proposal

![vicase-minipc-display-realview](https://cloud.githubusercontent.com/assets/7278716/16253777/f1abc8a4-386b-11e6-9fdd-bfafdf7f506e.jpg)

**Comment by bcaswelch** [comment]
# Problem

The splash screen on http://www.villagetronic.com is still promoting the old Indiegogo campaign.

# Proposal for text for new screen

## Title: Everything New

### New ViPC

We are developing a high end portable Workstation PC, ready for order in summer this year. Based on second generation ViCase. Key features:
- The first Open Modular Mini PC Workstation
- Small & portable Mini PC design.
- Fluid Form Factor enable changing a Mini PC to All-In-One and vice versa
- Many CPU choices up to i7-6700
- Ultra fast disks with PCI Express NVME boot disks and SSD RAID data disks
- Many GPU choices up to K1200
- Many ready build models available or you can go fully custom build
- Very quiet 3 chamber airflow design, yet cooling capacity of 300W
- Easy upgrading, easy maintenance, easy repair.
- Build in Surge Protection, EMI Filter and Power Distribution for displays 

### New ViCase

ViCase 2G is a high end Workstation DIY PC Case. Key Features:
- Very high quality. CNC cut from aircraft grade aluminum, sandblasted and anodized
- 10 Year Warranty
- Compatible to Desktop Standard Parts, e.g. Mini ITX Mainboard and Flex ATX PSU
- Optimized three chamber airflow design with cooling capacity 300W
- Double Bracket, Low Profile, 75W PCI Express Slot
- Small, portable dimensions: 245 x 245 x 105 mm³ = 6 Liters
- Washable magnetic quick snap filters
- Build in Surge Protection, EMI Filter and Power Distribution for displays 
- Easy access from all sides during assembly via removable sides

### New V² Community Development Site
- Anyone can file Bug Reports or Feature Ideas via V²Flow
- Anyone can improve articles or even contribute complete new ones via V²Flow
- We are currently focusing all our efforts on two sections of the V²Site
  - V²Plan
  - V²DIY

### New V² Community Stores

### A new page every day

We are now publishing a new page every day. We will announce it on our Facebook Page. Your feedback is most welcome.

**Comment by VillageHubertChen** [comment]
# Final Image

![vicase-minipc-display](https://cloud.githubusercontent.com/assets/7278716/16288935/55bde9c8-3923-11e6-8efa-52b2f772d223.jpg)

**Note:** Path on internal server: `\\MINI-PC\Users\Mini\Documents\Marketing\ViCase-ViPC`

**Comment by VillageHubertChen** [comment]
# Idea for screen shot from Georg:

3D App that shows the wire frame and the photo realistic rendering on one screen shot. Appealing to CAD engineers and designers.

**Comment by VillageHubertChen** [comment]
@bcaswelch 

Please finish editing your copy of the text we will publish on the home page.
